detekt/defs.bzl

```diff
@@ -8,6 +8,7 @@ _ATTRS = {
         executable = True,
         cfg = "exec",
     ),
+    "_default_baseline": attr.string(default = "detekt_baseline.xml"),
```

Comment:
What's the motivation behind having this as an attribute?

Reply:
moved it to the body of rule implementation